cli/file_fetcher.rs

```diff
@@ -424,9 +425,13 @@ impl SourceFileFetcher {
     let fake_filepath = PathBuf::from(module_url.path());
     let (media_type, charset) = map_content_type(
       &fake_filepath,
-      headers.get("content-type").map(|e| e.as_str()),
+      headers
+        .get("content-type")
+        .map(|e| e.first().unwrap().as_str()),
```

Comment:
`unwrap()` will cause panic if `content-type` header is not present

Reply:
thank you for your review
I updated it

same as present implement, 
if the key is not present, it returns `None` 
if the key has no value, it returns empty `String/str`

Reply:
@tokiedokie can you map no value to `None` instead of empty `String`? I think it might the logic in `map_content_type`. Otherwise LGTM

Reply:
@bartlomieju I'm done